dev.py

```diff
@@ -0,0 +1,4 @@
+from app import create_app
+
+if __name__ == "__main__":
```

Comment:
what's the purpose of this file?  do you run this instead of `python manage.py run`?